B.READ/B.READ/Sources/Presentation/Search/View/Main/SearchView.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  SearchView.swift
+//  B.READ
+//
+//  Created by 신승재 on 5/11/25.
+//
+
+import SwiftUI
+
+// MARK: - (S)SearchView
+struct SearchView: View {
+  @ObservedObject var viewModel: SearchViewModel
+  @State private var isSearchFocused: Bool = false
+  private let layoutSize: CGFloat = 16
+  
+  var body: some View {
+    VStack(alignment: .center, spacing: layoutSize) {
+      if !isSearchFocused {
+        logoView
+          .transition(.opacity)
+      }
+      
+      // 검색창은 한 개만 존재해야함
+      searchBarSection
+        .padding(.top, !isSearchFocused ? 0 : layoutSize)
+      
+      if isSearchFocused {
+        // TODO : DummyData, 추후 뷰모델 연결 필요
+        RecentSearchView(keywords: ["Test", "Test", "Test", "Test1", "Test3"])
+          .transition(.opacity)
+        Spacer()
```

Comment:
```suggestion
.frame(maxHeight: .infinity, alignment: .top)
```
p1) Spacer()사용을 지양하기로 했으니 이렇게 쓰는것이 나을 것 같습니당당

Reply:
넵 수정 완료 후, 잘 작동하는지 테스트 완료했습니다!